tools/scripts/utils/on-chain-utils/fetchAnchorMetadata.ts

```diff
@@ -205,12 +207,21 @@ async function fetchEVMAnchorMetadata(
       };
     }, {} as Record<string, string>);
 
+  const { timestamp: creationTimestamp } = await client.getBlock({
+    blockNumber: BigInt(anchorDeploymentBlock[typedChainId][anchorAddress]),
```

Comment:
Please include checks for `undefined` or `null` in the following instances:

- `anchorDeploymentBlock[typedChainId]` could potentially be `undefined`.
- Similarly, `anchorDeploymentBlock[typedChainId][anchorAddress]` might also be `undefined`.

Reply:
handle this by reject the function when the value is undefined!